public/js/app.js

```diff
@@ -23,10 +23,10 @@ projectMigrationApp.run(function($rootScope) {
     };
   } else {
     $rootScope.urls = {
-      'projectsUrl': '/projects.json',
-      'migrationsUrl': '/migrations',
-      'migratedUrl': '/migrated',
-      'projectUrl': '/projects/'
+      'projectsUrl': '/ctools-project-migration/projects.json',
+      'migrationsUrl': '/ctools-project-migration/migrations',
+      'migratedUrl': '/ctools-project-migration/migrated',
+      'projectUrl': '/ctools-project-migration/projects/'
```

Comment:
can we make `ctools-project-migration` as a constant instead of repeating if possible?